admin-dev/themes/new-theme/scss/pages/_orders_view.scss

```diff
@@ -0,0 +1,72 @@
+#order-view-page {
+  @media not print {
+    .card--details {
+      border: none;
+      box-shadow: none;
+      .card-body {
+        padding: 0;
+      }
+    }
+  }
+
+  @media print {
+    button, select {
+      display: none;
+    }
+
+    a {
+      text-decoration: none;
+      color: #000000;
+    }
+
+    .left-column,
+    .right-column {
+      flex: 0 0 100%;
+      max-width: 100%;
+      width: 100%;
+
+      .card, .tab-pane {
+        page-break-inside: avoid;
+      }
+    }
+
+    .tab-content {
+      padding: 0;
+      background-color: inherit;
+      .tab-pane {
+        opacity: 1;
+      }
+    }
+
+    .card--message {
+      form {
+        display: none;
+      }
+    }
+
+    .print-title {
+      .print-title__id {
+        color: #8a8a8a;
+      }
+    }
+  }
+}
+
+.adminorders {
+  @media print {
+    .header-toolbar {
```

Comment:
What's the use of this since all `.header-toolbar` seem to have the `d-print-none` class anyway

Reply:
Probably not, we see the toolbar if we don't do this, I'll investigate a bit :D

Reply:
it just seems redundant

Reply:
Ah no ok you didn't understand the purpose of that style, the header-toolbar doesnt have the d-print-none class because we need to see it on the first page to get the main title, but it is in `position: fixed` by default, we need to avoid this header bar to be seen on every print pages so we set it to `position: initial`

Reply:
But yet in `page_header_toolbar.tpl` it has `d-print-none`